File: PMS2/ClientApp/src/app/class/Portfolio.ts
import { Trade } from "./Trade";

export class Portfolio {
  portfolioId: number;
  name: string;
  portfolioTypeId?: number;
  Type?: string;
  TypeId?: string;
  InitialAmount?: number;
  Positions?: number;
  positionValue?: number;
  profit?: number;
  NumberOfTrades?: number;
  accountValue?: number;
  percentageGain?: number;
  openPosition?: Trade[];
  completedTrades?: Trade[];
}

File: PMS2/Viewmodel/Portfolio.cs
namespace PMS.Viewmodel
{
    public class Portfolio
    {
        public int PortfolioId { get; set; }
        public string Name { get; set; }
        public int PortfolioTypeId { get; set; }
        public long InitialAmount { get; set; }
        public int? Position { get; set; }
    }
}
